__tests__/screens/FeedScreen/HeaderFeed.test.js

```diff
@@ -51,7 +53,7 @@ describe('Header feed should run correctly', () => {
     expect(getAllByTestId('full')).toHaveLength(1);
   });
 
-  it('props isBackButton should have back button', () => {
+  it('props isBackButton not from feeds should have back button', () => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test description has been updated to specify that it's checking for the presence of a back button when `isBackButton` is true and the source is not from feeds. This makes the test case more specific and clear about what it's testing.
```diff
- it('props isBackButton should have back button', () => {
+ it('props isBackButton not from feeds should have back button', () => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->